cmd/gokr-packer/write.go

```diff
@@ -178,6 +180,70 @@ var (
 	}
 )
 
+type kernelManifest struct {
```

Comment:
Can you rename this type to deviceManifest for consistency with the flag name please?